sdk/src/main/java/com/kin/ecosystem/widget/TouchIndicatorIcon.java

```diff
@@ -73,8 +73,7 @@ private void setIcon(final Drawable icon) {
 		icon.setBounds(0, 0, iconSize, iconSize);
 		drawables[0] = icon;
 		Drawable touchIndicator = ContextCompat.getDrawable(getContext(), R.drawable.kinecosystem_info_red_dot);
-		final int left = icon.getIntrinsicWidth() - indicatorRadius;
-		touchIndicator.setBounds(left, 0, indicatorRadius + left, indicatorRadius);
+		touchIndicator.setBounds(0, 0, indicatorRadius, indicatorRadius);
```

Comment:
is this to make the click area bigger?

Reply:
it's to set it left top with the correct margins